sql/src/test/java/io/crate/integrationtests/FulltextAnalyzerResolverTest.java

```diff
@@ -329,7 +329,7 @@ public void testBuiltInTokenFilters() throws Exception {
                "pattern_replace, persian_normalization, porter_stem, reverse, " +
                "russian_stem, scandinavian_folding, scandinavian_normalization, serbian_normalization, " +
                "shingle, snowball, sorani_normalization, standard, stemmer, stemmer_override, " +
-               "stop, synonym, trim, truncate, unique, uppercase, word_delimiter"));
+               "stop, synonym, trim, truncate, unique, uppercase, word_delimiter, word_delimiter_graph"));
```

Comment:
@mfussenegger this token filter is experimental: https://www.elastic.co/guide/en/elasticsearch/reference/5.4/analysis-word-delimiter-graph-tokenfilter.html